doc/release-notes.md

```diff
@@ -19,13 +19,27 @@ How to Upgrade
 --------------
 
 If you are running an older version, shut it down. Wait until it has completely
-shut down (which might take a few minutes for older versions), then run the
-installer (on Windows) or just copy over /Applications/Dash-Qt (on Mac) or
-dashd/dash-qt (on Linux). If you upgrade after DIP0003 activation and you were
-using version < 0.13 you will have to reindex (start with -reindex-chainstate
+shut down (which might take a few minutes in some cases), then run the
+installer (on Windows) or just copy over `/Applications/Dash-Qt` (on Mac)
+or `dashd`/`dash-qt` (on Linux).
+
+Upgrading directly from a version of Dash Core that has reached its EOL is
+possible, but it might take some time if the data directory needs to be migrated. Old
+wallet versions of Dash Core are generally supported. If you upgrade after DIP0003 activation
+and you were using version < 0.13 you will have to reindex (start with -reindex-chainstate
```

Comment:
20419 this should be in a different PR if done

Reply:
Can't make it a different PR, minimum SDK version is a prerequisite. Will move documentation update to a new PR. Marking as partial